test/e2e/validatingwebhook/validatingwebhook_utils.go

```diff
@@ -0,0 +1,125 @@
+/*
+    Copyright (C) 2020 Accurics, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+package validatingwebhook
```

Comment:
this package name is confusing. all of the functions inside are utilities that deal with kubernetes and kind clusters. so naming the package validatingwebhook is a bit misleading. maybe just call it `util` or `kube_util`